spec/requests/works_spec.rb

```diff
@@ -265,7 +265,7 @@
       end
 
       it 'displays the work' do
-        post "/collections/#{collection.id}/works", params: { work: work_params }
+        post "/collections/#{collection.id}/works", params: { work: work_params, commit: 'Deposit' }
```

Comment:
making all of these have "Deposit", means we're no longer testing the draft branch.

Reply:
we definitely will need a new test or two to verify the ability to save a draft without validation (which currently doesn't work in this draft PR by the way, as we have validation at the model level as well, preventing the save)...

were some of these tests actually exercising the draft button before?

Reply:
Added a new test specifically to show that you can save a draft without the normal validations.  Switched all other tests to explicitly submit for both works and collections.